templates/zerver/help/send-a-stream-message.md

```diff
@@ -1,31 +1,31 @@
 ## Sending a new stream message
 
-![New stream message](/static/images/help/new-stream-message.png)
+You can follow the following steps to send a new stream message.
 
-To submit a new stream message, first click the **New stream message**
-button located at the bottom of your screen.
+1. Click the **New stream message** button(similar to the button shown above)
+located at the bottom of your screen. The message box, similar to the one shown below will appear.
+  ![New stream message](/static/images/help/new-stream.png)
 
-![New stream message](/static/images/help/new-stream.png)
-
-A box similar to the one shown in the above image will appear.
+2. In the **Stream** field, input the name of the stream you want your message
+to be located in. When you begin typing a stream name, autocomplete suggestions
+will appear.
 
-In the **Stream** field, input the name of the stream you want your
-message to be located in. When you begin typing a stream name,
-autocomplete suggestions will appear.
+    !!! warn ""
+        If you attempt to input a nonexistent stream name, an error
+        message will appear.
 
-![Stream does not exist](/static/images/help/stream-none.png)
+    ![Stream does not exist](/static/images/help/stream-none.png)
 
-If you attempt to input a nonexistent stream name, an error message
-will appear.
+3. Enter the topic for your message in the **Topic** field. Your topic name can
+be no longer than 52 characters.
 
-Once you have entered your stream name, enter the topic for your
-message in the **Topic** field. Your topic name can be no longer than
-52 characters.
+4. Enter your message in the box labeled **Compose your message here...**, also
+known as the messaging box.
 
-Finally, enter your message in the box labeled **Compose your message
-here...** or messaging box. You can now send your message to the
-specified stream under the specified topic.
+5. Press the **send** button if the **Press Enter to send** checkbox is not
+selected. If it is, press enter to send your message.
```

Comment:
You can make this more clearer by clarifying you press the "Enter or return key" to send your message.